src/Conf.ml

```diff
@@ -164,6 +168,14 @@ end = struct
     store := Pack opt :: !store ;
     opt
 
+  let bool_no_cmdline ~name ~update =
```

Comment:
Would it be bad to allow `--disable` on the command line, so that we can document it's intended use as attributes and in config files in the same place with the other options?

Reply:
done